dashboard/app/models/concerns/partial_registration.rb

```diff
@@ -22,7 +22,6 @@ def new_from_partial_registration(session)
           user.send("#{param}=", cache.read(cache_key)) if cache
         end
         yield user if block_given?
-        user.valid?
```

Comment:
Note on this: calling `user.valid?` causes model validations to run, so the user would see errors on initial page load for the finish sign up form. Looking through tests, it does not appear that we need to run validations yet, but please let me know if I'm missing something!

Reply:
Note, this code _does_ run for both old and new flows, so this might be reponsible for the validation errors showing up immediately after callback in the old flow.  If that's a behavior we want to preserve :man_shrugging: I'd double-check it before removing this.

Reply:
You're right! Thanks for catching that. I think we would want to preserve the old behavior, so maybe this line should change to:
`user.valid? unless SignUpTracking.new_sign_up_experience?(session)`